src/Properties/Email.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace GingerPluginSdk\Properties;
+
+use GingerPluginSdk\Bases\BaseField;
+use GingerPluginSdk\Exceptions\OutOfPatternException;
+use GingerPluginSdk\Interfaces\ValidateFieldsInterface;
+
+final class Email extends BaseField implements ValidateFieldsInterface
+{
+    /**
+     * @throws \GingerPluginSdk\Exceptions\OutOfPatternException
+     */
+    public function validate($value)
+    {
+        if (!filter_var($value, FILTER_VALIDATE_EMAIL)) {
+            throw new OutOfPatternException($this->getPropertyName(), ['test@mail.com', 'another@gmail.com']);
```

Comment:
I assume the test email's are not supposed to be here?

Reply:
Is a just example, to provide the best experience with a library for devs. To make response of Exception with more payload